src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: myFont;
+  src: url(./fonts/Avenir.ttc);
+
+}
+
+body {
+  font-family: myFont, sans-serif;
+  color: #3d4e61;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
+  margin: 8px;
+  padding: 0;
+}
+
+.search {
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  height: 68px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  margin: 20px auto 0;
+}
```

Comment:
As I've described below, we don't need to separate styles and give unnecessary wrappers, so this class should only have indents or `position: relative' if the image is conditionally positioned with an absolute. You can also set the image as a background

Reply:
moved to search__input class